src/pages/dashboard/forms/token-deposit-form.tsx

```diff
@@ -95,7 +95,7 @@ const TokenDepositForm = (props: Props) => {
 
       <div className={styles.formContent}>
         <div className={styles.inputWrapper}>
-          <Input type="number" autosize value={inputValue} onChange={(e) => setInputValue(e.target.value)} autoFocus />
+          <AutosizeInput type="number" value={inputValue} onChange={(e) => setInputValue(e.target.value)} autoFocus />
```

Comment:
I would keep the old component name - `<Input />`. We are using `react-input-autosize` library under the hood but from the outside it's just input.